src/fieldset/index.jsx

```diff
@@ -177,7 +179,7 @@ function Field({
   }, [fieldValue]);
 
   if (formatHint) {
-    hint = formatHint(hint);
+    hint = formatHint({ name, prefix, hint });
```

Comment:
This bit doesn't seem to fit anywhere with the rest of the PR and looks like it will be a breaking change.

Reply:
This is the related PR: https://github.com/UKHomeOffice/asl-pages/pull/1264

This is the only place formatHint is used currently: 
https://github.com/UKHomeOffice/asl-pages/blob/master/pages/rops/update/formatters/index.jsx
and none of those actually need fixing because they don't use the `hint` param.